addons/account/wizard/account_move_send.py

```diff
@@ -776,7 +776,7 @@ def action_send_and_print(self, force_synchronous=False, allow_fallback_pdf=Fals
             raise UserError(_('Please select a mail template to send multiple invoices.'))
 
         force_synchronous = force_synchronous or self.checkbox_download
-        process_later = self.mode == 'invoice_multi' and not force_synchronous
+        process_later = force_process_later or (self.mode == 'invoice_multi' and not force_synchronous)
```

Comment:
Maybe we could trick a bit here:
```py
def action_send_and_print(self, force_synchronous=None, allow_fallback_pdf=False, **kwargs):
```
```py
if force_synchronous is None:
    process_later = self.mode == 'invoice_multi'
else:
    process_later = not force_synchronous
```
@clbr-odoo I summon you!

Reply:
```python
if not force_synchronous:
    process_later = self.mode == 'invoice_multi'
else:
    process_later = False  # process now !
```
Your trick should work, and make sense since "process_later" is equivalent to "asynchronous".
Could be slightly easier to read ?